app/src/commonMain/kotlin/io/github/cgr/app/Entry.kt

```diff
@@ -3,13 +3,19 @@
 package io.github.cgr.app
 
 import io.github.cgr.app.io.readProgram
+import io.github.cgr.app.objects.BaseLabel
+import io.github.cgr.app.objects.BaseOpcode
+import io.github.cgr.app.objects.ConstantLabel
+import io.github.cgr.app.objects.Opcode
 import kotlin.js.ExperimentalJsExport
 import kotlin.js.JsExport
 import kotlin.jvm.JvmName
 
 @JsExport
 @OptIn(ExperimentalJsExport::class)
 fun executeFile(file: ByteArray) {
+    BaseOpcode.values().forEach(Opcode::register)
+    BaseLabel.values().forEach(ConstantLabel::register)
```

Comment:
This is significantly less cursed than what I thought was going on based on the description you gave in TNB.